docs/src/templates/pages/documentation/css/index.hbs

```diff
@@ -1,141 +1,59 @@
 ---
 title: CSS Principles
 section-title: CSS Documentation
-description: Our commitment to creating an accessible platform
+description: The principles around how we use CSS at Just Eat
 docs: true
 
 navgroup: documentation
 navsub: docs-css
 navactive: css-principles
-
-wip: true
 ---
 
-## Note – Page needs updating with Global in mind! (currently international CSS docs here)
-
 The aim when writing CSS on any platform is to ensure it is well structured, maintainable and flexible.
 
-To help us do that at JUST EAT, we have adopted [Sass](#sass), a CSS preprocessor which allows us to use many extra features beyond the capabilities of Vanilla CSS.
+To help us do that at Just Eat, we use [Sass](#sass) and PostCSS, which allows us to utilise extra CSS processing features beyond the capabilities of Vanilla CSS.
 
-This section of the documentation aims to explain how we use CSS and Sass at JUST EAT, as well as covering the methodologies and best practices that have helped to guide our way of thinking.
+Below, we'll aim to explain how we use CSS, Sass and PostCSS on our projects, as well as covering the methodologies and best practices that have helped to guide our way of thinking.
 
 
-<a name="philosophy"></a>
-
-## Philosophy
+<a name="philosophy"></a><h2 class="sg-sectionHeading">Philosophy</h2>
 
 When writing CSS for any website, it is important to write styles that are both maintainable and flexible, so that they are easy to extend and can be reused in the future.
 
-There are many methodologies that support this way of thinking, such as [SMACSS](https://smacss.com/), [Atomic Design](http://bradfrost.com/blog/post/atomic-web-design/) and [Object Oriented CSS](http://www.smashingmagazine.com/2011/12/an-introduction-to-object-oriented-css-oocss/).  These all share a lot of common ground in their opinions on constructing good CSS; essentially that when developing styles you should think about how you can write small reusable components that can be easily extended later.
-
-We encourage this way of thinking at JUST EAT and our Sass is [structured to reflect this](#structure), with base styles defined globally and individual components and modules that extend upon these styles.
+There are many CSS methodologies that support this way of thinking, such as [SMACSS](https://smacss.com/), [Atomic Design](http://bradfrost.com/blog/post/atomic-web-design/)) and [ITCSS](https://www.xfive.co/blog/itcss-scalable-maintainable-css-architecture/).  These all share a lot of common ground whn it comes to constructing good CSS; one of main themes being that you should always think about how you can write small reusable components that can be easily extended later.
```

Comment:
"whn" -> "when"
And a comma between small and reusable while you're at it, please!